src/mixins/trackHelper.js

```diff
@@ -31,6 +31,10 @@ export var getTrackCSS = function(spec) {
     msTransform: 'translateX(' + spec.left + 'px)'
   };
 
+  if (!-[1,]) {
```

Comment:
@vramana ok, I've pushed the fix, maybe that's why @sorrycc went for `for` loop.
`map` doesn't iterate with empty array unless it's prefilled.

It might be related to this: https://github.com/es-shims/es5-shim/issues/190 I just added a quick fix for this case.